R/as_html.R

```diff
@@ -101,23 +107,56 @@ as_html <- function(x,
   ## special casing hax for top_left. We probably want to do this better someday
   cells[1:nrh, 1] <- mapply(
     FUN = function(x, algn) {
-      tags$th(x, class = class_th, style = "white-space:pre;")
+      tags$th(x, class = class_th, style = "white-space: pre;")
     },
     x = mat$strings[1:nrh, 1],
     algn = mat$aligns[1:nrh, 1],
     SIMPLIFY = FALSE
   )
 
-  # indent row names
+  if (header_sep_line) {
+    cells[nrh][[1]] <- htmltools::tagAppendAttributes(
+      cells[nrh, 1][[1]],
+      style = "border-bottom: 1px solid black;"
+    )
```

Comment:
I am a bit doubtful about this... I think they should be all the same (horizontal lines). Maybe, we could add a swap between the two types

Reply:
The main problem with that is that titles/footers don't exist within the HTML table structure (i.e. aren't rows in the table), so we can't fill in the table "border" like we do in the table to create that line. We would have to figure out a different method that works for both (if one exists)

Reply:
So also, the header cannot have the hsep_line? Sorry if I am slow here. did not use too much HTML lately :D

Reply:
That's correct. Don't worry about it, the HTML formatting stuff is a bit strange!